src/main/java/edu/harvard/iq/dataverse/DataFileServiceBean.java

```diff
@@ -1366,7 +1366,10 @@ public Embargo findEmbargo(Long id) {
         DataFile d = find(id);
         return d.getEmbargo();
     }
-    
+
+    public boolean isActivelyRetended(FileMetadata fm) {
```

Comment:
maybe isExpired? isNoLongerRetained()? (I'm not sure retended is a word and actively rentended/under retension could be confusing as, unlike embargo, retention is the period it's available and the file is not /is no longer retained when this method is true.

Reply:
I think the word 'expired' is better than 'no longer', so if I understand you correctly I have to implement  the following:

- While it is 'under retention' the file is  available and in a  'retained' state. 
- After the retention period, the file is unavailable and that is the 'retentionExpired' state. 
- The data that is being set for the retention is then the retention 'end-date' and the file is available on that date but not after that.